docroot/modules/custom/foia_webform/foia_webform.module

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * FOIA Webform module.
+ */
+
+use Drupal\webform\WebformInterface;
+
+/**
+ * Programmatically add the foia_email handler to new webforms.
+ */
+function foia_webform_webform_insert(WebformInterface $webform) {
+  $webform_handler_manager = \Drupal::service('plugin.manager.webform.handler');
+  $webform_handler = $webform_handler_manager->createInstance('foia_email');
+
+  // @todo: Add Submission Email if we add Agency Component entity reference.
+  // Set default configuration.
+  $webform_handler->setConfiguration([
+    'id' => 'foia_email',
+    'label' => 'FOIA Email',
+    'handler_id' => 'foia_email',
+    'status' => TRUE,
+    'weight' => 0,
+    'settings' => [
+      'states' => ['completed'],
+      'to_mail' => '',
+      'to_options' => [],
+      'cc_mail' => '',
+      'cc_options' => [],
+      'bcc_mail' => '',
+      'bcc_options' => [],
+      'from_mail' => 'default',
+      'from_options' => [],
+      'from_name' => 'default',
+      'subject' => '[webform_submission:values:subject:value]',
+      'body' => '[webform_submission:values:message:value]',
+      'excluded_elements' => [],
+      'html' => TRUE,
+      'attachments' => FALSE,
```

Comment:
Maybe this should be `TRUE`?

It looks like webform is supposed to handle attachments out of the box: http://cgit.drupalcode.org/webform/tree/src/Plugin/WebformHandler/EmailWebformHandler.php#n780